consumergroup.go

```diff
@@ -73,15 +70,23 @@ type ConsumerGroupConfig struct {
 	// must not be empty.
 	Brokers []string
 
-	// An dialer used to open connections to the kafka server. This field is
-	// optional, if nil, the default dialer is used instead.
-	Dialer *Dialer
-
 	// Topics is the list of topics that will be consumed by this group.  It
 	// will usually have a single value, but it is permitted to have multiple
 	// for more complex use cases.
 	Topics []string
 
+	// A transport used to send messages to kafka clusters.
+	//
+	// If nil, a Transport will be created used the provided ClientID or the DefaultClientID
+	// if not provided.
+	Transport RoundTripper
```

Comment:
Would it be possible to use `kafka.DefaultTransport` instead of creating a new transport when the program doesn't provide one?

Reply:
Need to set the client id for the consumer group; Let's see what happens if multiple consumers use the same ID. 
Otherwise document users are expected to create a unique transport for the Reader.